manuscript.md

```diff
@@ -36,6 +36,22 @@
 
 ## What is a species interaction?
 
+The coexistence of species does not necessarily preclude that they will
+interact [@Blanchet2020CooNot], however (at least for *direct* interactions) 
+we would require species to co-occur in space and time [@Walther2010ComEco].  
+The intrinsically diverse and complex nature of species interactions 
+[something we could address in the glossary perhaps?] makes 
+it difficult to accurately describe and collect data that are accurate and 
+comprehensive. However, a *mechanistic* understanding of what 
+determines if and how species will interact can be leveraged when trying to 
+predict interactions when data are incomplete or missing.
+
+* Do we continue on a tangent about the 'ecological aspect' - so in a way
```

Comment:
This should be the first sentence of the ¶, in fact, where you list different types of interactions to make sure that people don't get the idea that the paper is only food webs, or only mutualisms